__init__.py

```diff
@@ -56,9 +64,12 @@ def handle_intent(self, message):
         except Exception as e:
             LOGGER.error("Error: {0}".format(e))
 
+    def handle_stop(self, message):
+        self.stop()
+        self.speak_dialog('npr.news.stop')
```

Comment:
I moved it to handle_stop because I didn't want this dialog to be said when self.stop() is called from handle_intent(). But its not a big deal. I can move it back to stop if you confirm.